### DISCUSSION TITLE
Pimcore 11:  profile and logout button auto remove after click on csv export button in custom report

### DISCUSSION THREAD

**Discussion by sandyHM** [pimcore/pimcore#17099] [Q&A / Support] [answered]
Pimcore profile and logout button auto remove after click on csv export button in custom report
![screenshot-nimbusweb me-2024 05 29-12_25_30](https://github.com/pimcore/pimcore/assets/107986459/998ea98c-c6ec-412d-83df-d11de3b254a3)

**Comment by kingjia90** [comment]
Thank you for reporting, i could reproduce it , opened an issue https://github.com/pimcore/admin-ui-classic-bundle/issues/541

**Reply by kingjia90** [reply, marked as answer]
Ah wait, it's resolved by https://github.com/pimcore/admin-ui-classic-bundle/pull/522 in the upcoming `admin-ui-classic-bundle` `1.4.3` but it is not tagged yet, planned for next week, on 4th June

**Reply by sandyHM** [reply]
Hi @kingjia90 
thank you for update.

**Comment by sandyHM** [comment]
fixed in https://github.com/pimcore/admin-ui-classic-bundle/pull/522
